apps/agora/app/src/config/config.json

```diff
@@ -5,5 +5,6 @@
   "dataUpdatedOn": "yyyy-mm-dd",
   "environment": "development",
   "googleTagManagerId": "",
-  "ssrApiUrl": "http://agora-api:3333/v1"
-}
\ No newline at end of file
+  "ssrApiUrl": "http://agora-api:3333/v1",
+  "rollbarToken": "e788198867474855a996485580b08d03"
```

Comment:
Should this be in a gitignore?

Reply:
This is the file that the dev server uses, so it shouldn't be added to `.gitignore`.